contentcuration/contentcuration/utils/nodes.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+from django.conf import settings
+from django.core.files import File as DjFile
+from django.core.files.storage import default_storage
+from django_s3_storage.storage import S3Error
+
+from contentcuration.models import Language, User, ContentNode, FormatPreset, generate_object_storage_name, File
+
+def map_files_to_node(user, node, data):
+    """
+    Generate files that reference the content node.
+    """
+    if settings.DEBUG:
+        # assert that our parameters match expected values
+        assert isinstance(user, User)
+        assert isinstance(node, ContentNode)
+        assert isinstance(data, list)
```

Comment:
@kollivier lemme know your opinion of this convention. I added asserts like a previous PR, but this time I wrapped them around a `settings.DEBUG` call. That should prevent us from encountering these asserts on prod, but hopefully we can still make use of them in dev mode.